File: app/src/main/java/com/coinninja/coinkeeper/cn/wallet/SeedWordGenerator.java
package com.coinninja.coinkeeper.cn.wallet;

import com.coinninja.bindings.Libbitcoin;
import com.coinninja.coinkeeper.di.interfaces.CoinkeeperApplicationScope;

import javax.inject.Inject;

@CoinkeeperApplicationScope
public class SeedWordGenerator {

    @Inject
    public SeedWordGenerator() {
    }

    public String[] generate() {
        return new Libbitcoin().getSeedWords();
    }

}

File: app/src/main/java/com/coinninja/coinkeeper/cn/wallet/SeedWordGenerator.kt
package com.coinninja.coinkeeper.cn.wallet

import app.dropbit.annotations.Mockable
import com.coinninja.bindings.Libbitcoin
import com.coinninja.coinkeeper.di.interfaces.CoinkeeperApplicationScope

import javax.inject.Inject

@Mockable
@CoinkeeperApplicationScope
class SeedWordGenerator @Inject constructor() {

    fun generate(): Array<String> {
        return Libbitcoin().createSeedWords
    }

}
